demos/controlgroup/splitbutton.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>jQuery UI Controlgroup - Split Button</title>
+	<link rel="stylesheet" href="../../themes/base/all.css">
+	<script src="../../external/jquery/jquery.js"></script>
+	<script src="../../ui/core.js"></script>
+	<script src="../../ui/widget.js"></script>
+	<script src="../../ui/button.js"></script>
+	<script src="../../ui/checkboxradio.js"></script>
+	<script src="../../ui/menu.js"></script>
+	<script src="../../ui/position.js"></script>
+	<script src="../../ui/selectmenu.js"></script>
+	<script src="../../ui/controlgroup.js"></script>
+	<link rel="stylesheet" href="../demos.css">
+	<script>
+	$(function() {
+		$( "select" ).selectmenu({
+			classes: {
+				"ui-selectmenu-button": "ui-button-icon-only"
```

Comment:
Im not sure why you would consider it "hacky" to add a class with the classes option?

> There's a button option for this after all.

I'm not sure what you mean a button option on what? selectmenu does not have a button option? 
if your referring to using `_drawButton` This is not a documented extension point and is far from trivial, there is `_renderButtonItem` but this is also not documented and this only renders the text span within the button not the button it self.